packages/terra-markdown/tests/wdio/markdown-spec.js

```diff
@@ -0,0 +1,23 @@
+const viewports = Terra.viewports('large');
+
+describe('Markdown', () => {
+  describe('Default', () => {
```

Comment:
Since we are only testing one viewport, lets set the viewport size once instead of triggering a resize for each test step. This will speed up test execution

```diff
- const viewports = Terra.viewports('large');

describe('Markdown', () => {
+   before(() => browser.setViewportSize(Terra.viewports('large')[0]));
     describe('Default', () => {
-        Terra.should.beAccessible({ viewports });
-        Terra.should.matchScreenshot({ viewports });
+        Terra.should.beAccessible();
+        Terra.should.matchScreenshot();
....
```